kubectl-cloudflow/docker/client.go

```diff
@@ -136,3 +156,50 @@ func imageMatchesNameAndTag(image types.ImageSummary, imageNameAndTag string) bo
 	}
 	return false
 }
+
+// GetStreamletDescriptorsForImage extracts the configuration of a Cloudflow label from a docker image
+// and the image with digest in a struct
+func GetStreamletDescriptorsForImage(cli *client.Client, imageName string) (domain.CloudflowStreamletDescriptorsDigestPair, string) {
```

Comment:
I know this has been done before but I don't really think that `GetStreamletDescriptorsForImage` should be part of the `docker` package. The same goes for `GetCloudflowApplicationDescriptor` like methods. not sure when we should clean up some of these structures. Since the cli will get more code, it's probably good to add a ticket for this.

Reply:
👍